scoring_engine/db.py

```diff
@@ -1,56 +1,12 @@
+import bcrypt
 from sqlalchemy import create_engine
 from sqlalchemy.orm import scoped_session, sessionmaker
 
-import bcrypt
-
-from scoring_engine.db_not_connected import DBNotConnected
-
 from scoring_engine.engine.config import config
 
-db_salt = bcrypt.gensalt()
-
-
-class DB(object):
-    def __init__(self):
-        self.connected = False
-        self.db_uri = config.db_uri
 
-    def connect(self):
-        self.connected = True
-        self.engine = create_engine(
-            self.db_uri,
-            convert_unicode=True,
-        )
-        self.session = scoped_session(sessionmaker(
-            bind=self.engine,
-            autocommit=False,
-            autoflush=True,
-            expire_on_commit=True,
-        ))
+engine = create_engine(config.db_uri, isolation_level="READ UNCOMMITTED")
```

Comment:
This line actually fixes the caching issue we found. I would think we would want this to be "READ COMMITTED", but unfortunately sqlite db doesn't support it. So maybe we create a mysql connection with "READ COMMITTED", and if we use sqlite, we use a different isolation level?